autopilot/contracts.go

```diff
@@ -1,16 +1,34 @@
 package autopilot
 
 import (
+	"time"
+
+	"gitlab.com/NebulousLabs/errors"
 	"go.sia.tech/renterd/internal/consensus"
+	rhpv2 "go.sia.tech/renterd/rhp/v2"
 	"go.sia.tech/renterd/worker"
 	"go.sia.tech/siad/types"
 	"golang.org/x/crypto/blake2b"
 )
 
+const contractSetName = "autopilot"
```

Comment:
So we are going with the contract set idea? Where we can pass a whole set for uploads/downloads?

Reply:
I'm personally not a big fan of contract sets here. I think we should simply fetch something like "active contracts" instead.

Reply:
I didn't really see the value at first but I think it's kind of nice because it puts the autopilot in the driver seat as it can control the contracts without having to implement quirky logic in the bus or communicate through a bunch of database fields.